html/changelogs/icantthinkofanameritenow

```diff
@@ -0,0 +1,4 @@
+author: Icantthinkofanameritenow
+
+changes: 
+-tweak: Replaces Advanced Energy Gun in protolathe with a modkit that can be used on an energy gun to convert it into an AEG. 
```

Comment:
Actually fun fact, despite people having spread that rumor for ages, it doesn't need 2 spaces before the -.